src/components/Lookup/index.js

```diff
@@ -236,6 +243,63 @@ class Lookup extends Component {
         }
     }
 
+    // -- mine
+
+    stopArrowScoll() {
+        if (this.scrollingTimer) {
+            clearInterval(this.scrollingTimer);
+        }
+    }
+
+    scrollTo(offset) {
+        const menu = this.menuRef.current.getRef();
+        menu.scrollTo(0, offset);
+    }
+
+    scrollBy(offset) {
+        const menu = this.menuRef.current.getRef();
+        menu.scrollBy(0, offset);
+    }
+
+    handleScrollUpArrowHover() {
+        this.stopArrowScoll();
+
+        const menu = this.menuRef.current.getRef();
+        this.scrollingTimer = setInterval(() => {
+            if (menu.scrollTop > 0) {
+                menu.scrollBy(0, -1);
+            } else {
+                this.stopArrowScoll();
+            }
+        }, 5);
```

Comment:
not a fan on `setInterval`, prefer to implement same behavior using only `setTimeout`, setInterval can cause serious memory leaks

Reply:
i changed `setInterval` by `setTimeout` already.